app/models/feedback.rb

```diff
@@ -1,9 +1,7 @@
 class Feedback < ApplicationRecord
   belongs_to :receiver
 
-  enum response: { gift_found: 0, gift_not_found: 1, closed: 2 }
-
-  validates :response, presence: true
+  validates_inclusion_of :fulfilled, in: [true, false]
```

Comment:
Prefer the new style validations validates :column, inclusion: value over validates_inclusion_of.